src/java.base/share/classes/java/util/Locale.java

```diff
@@ -794,16 +786,99 @@ public Locale(String language, String country) {
      * any syntactic checks on the input.
      * </ul>
      *
+     * @deprecated Locale constructors have been deprecated. See <a href="#ObtainingLocale">
+     * Obtaining a Locale</a> for other options.
+     *
      * @param language An ISO 639 alpha-2 or alpha-3 language code, or a language subtag
      * up to 8 characters in length.  See the {@code Locale} class description about
      * valid language values.
      * @throws    NullPointerException thrown if argument is null.
      * @since 1.4
      */
+    @Deprecated(since="19")
     public Locale(String language) {
         this(language, "", "");
     }
 
+    /**
+     * Obtains a locale from language, country and variant.
+     * This method normalizes the language value to lowercase and
+     * the country value to uppercase.
+     * @implNote
+     * <ul>
+     * <li>Obsolete ISO 639 codes ("iw", "ji", and "in") are mapped to
+     * their current forms. See <a href="#legacy_language_codes">Legacy language
+     * codes</a> for more information.
+     * <li>For backward compatibility reasons, this method does not make
```

Comment:
As `variant` cannot meet with bcp47 spec, it will have to accept ill-formed arguments. I think we will describe it here and suggest users use other means if they want stricter restrictions.

Reply:
sure, I was thinking of consistency with LocaleBuilder.

Reply:
Modified the method descriptions.

Reply:
I would have preferred to throw for invalid language and country, even if variant can't be validated.
Otherwise, it seems the descriptions on the @param tags is pointless and should just be omitted;
adding only a reference in the method javadoc.

Reply:
Note that `Locale.forLanguageTag()` won't throw an exception for a given ill-formed language tag. It's up to the user to select which one fits their needs. Considering Locale.of() are stop-gap replacements to the constructors, accepting ill-formed is the better choice, IMO.
As to the @param, I modified them to point to the descriptions in the class description.